pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +921,30 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAPrepareResourcesDuration tracks the duration of the DRA PrepareResources request.
+	DRAPrepareResourcesDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAPrepareResourcesDurationKey,
+			Help:           "Duration in seconds of the DRA PrepareResources request.",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
```

Comment:
@logicalhan: how is promotion of metrics to beta tracked? Does this need to be tracked in a KEP?

Open question: should or should not this metric use the driver name as label? I see that as a blocker that we should clarify before mergeing, so:

/hold

Reply:
We track promotion of metrics using the stability framework via our static analysis tooling (which runs on precommit against the entire codebase). If you attempt to upgrade the stability level, the list of beta metrics will grow and require an instrumentation audit on the promoted metric.

In alpha it is acceptable to add or remove labels. In beta, you can no longer remove labels, but adding them is okay. Once a metric is stable, it's API must not change into the future.

Reply:
Are you tracking metrices that remain in alpha or beta too long?

Reply:
No, metrics can stay in Alpha as long as they want. We do generate documentation for all kubernetes metrics (via our static analysis tooling) each release, so there is a track record of it if we wanted to enforce something, but I don't really see a need at the moment.

Reply:
/unhold